src/_guides/language/evolution.md

```diff
@@ -313,74 +371,74 @@ multiple versions of the Dart language.
 The compiler determines what version the code is targeting,
 and it interprets the code according to that version.
 
-Language versioning is important on the rare occasions
-when Dart introduces an incompatible feature like [null safety][].
-Code that used to compile cleanly before null safety
-(but perhaps crash at runtime)
-might no longer compile once null safety is enabled.
-Because migrating your apps and packages—and all the packages they depend on—to
-null safety might take a while,
-Dart uses language versioning to support
-using non-null-safe code alongside null-safe code.
-For an example of how an app or package can migrate to
-a new language version
-with an incompatible feature (null safety, for example), see 
+Language versioning becomes important on the rare occasions when Dart
+introduces an incompatible feature like [null safety][].
+When Dart introduces a breaking change, code that
+did compile might no longer compile.
+Language versioning allows you to set each library's language version
+to maintain compatibility.
+
+In the case of null safety, Dart SDKs 2.12 through 2.19 allowed you
+to _choose_ to update your code to use null safety.
+Dart uses language versioning to permit non-null-safe code to run
+alongside null-safe code.
+This decision enabled migration from non-null-safe to null-safe code.
+To review an example of how an app or package can migrate to a new
+language version with an incompatible feature, check out
 [Migrating to null safety](/null-safety/migration-guide).
 
-Each package has a default language version,
-equal to the `<major>.<minor>` part of the
-**lower SDK constraint** in the pubspec.
-For example, the following entry in a `pubspec.yaml` file
-indicates that this package uses the Dart 2.7 language version.
+Each package has a default language version equal to the **lower bound**
+of the SDK constraint in the `pubspec.yaml` file.
+
+**For example:** The following entry in a `pubspec.yaml` file
+indicates that this package uses the Dart 2.18 language version or later.
```

Comment:
This indicates it uses the 2.18 language version by default, not later. It can't use later language versions or else that would break the constraint.

```suggestion
indicates that this package uses the Dart 2.18 language version by default.
```